custom_components/cielo_home/switch.py

```diff
@@ -23,13 +24,21 @@ async def async_setup_entry(
         entities.append(
             CieloHomeSwitchPower(device, "Power", device.get_uniqueid() + "_power")
         )
+
         if device.get_is_appliance_is_freezepoin_display():
             entities.append(
                 CieloHomeSwitchFreezingPoint(
                     device, "Freezing Point", device.get_uniqueid() + "FreezingPoint"
                 )
             )
 
+        if device.get_device_type() == "BREEZ-MAX":
```

Comment:
you must check if the device is support follow-me, not if it's Breeze max. My device support it and i don't have a Breeze-Max.